### ISSUE TITLE
How to use this plugin?

### ISSUE THREAD

**Issue by polymake-pigeon** [zhucyi/project-tree#5]
How do I use this plugin?
I have installed the plugin but can't seem to make it do anything!
I press press ctrl+shift+p  but nothing happens.

**Comment by fbatarags** [comment]
Pressing Ctrl+Shift+P will open the prompt starting with > in VSCode, there type "Project Tree". The generated tree will be added to README.md file in project root.

**Comment by polymake-pigeon** [comment]
My project does not already have a README.md file, do I have to create one first?

**Comment by zhucyi** [comment]
If there is no README.md in the project directory, it will be automatically generated. If there is README.md, it will be appended to the end of the existing file.

**Comment by polymake-pigeon** [comment]
I have a project which already has a README.md and Ctrl+Shift+P creates the project tree inside it.
I have another project which does not have a README.md but Ctrl+Shift+P does not create one ,it seems to do nothing.
Where do I do Ctrl+Shift+P? In terminal? Does it matter where I put the cursor when I do Ctrl+Shift+P?

**Comment by zhucyi** [comment]
Is the screen focus not in vscode when the creation fails? Choose any file and follow the steps to try it out?

**Comment by polymake-pigeon** [comment]
Tried it again several times, README.md file is not generated.

**Comment by zhucyi** [comment]
Does vscode report wrong information?

**Comment by polymake-pigeon** [comment]
In the project which has no README.md file, vscode reports nothing.

I have confirmed that the project tree is created and put into the README.md file if it already exists, and when it does VSCode reports that the README.md file has been updated.  

I think this plugin is a good idea but I don't think this plugin is giving me exactly what I want, so I am not going to waste any more time with it.

Thanks for your help!
